settings.gradle

```diff
@@ -1,3 +1,4 @@
 rootProject.name = 'gradle-utils'
 
 include 'lazily-configured-mapping'
+include 'environment-variables'
```

Comment:
I'd add a section to the readme, even super brief, just showing how to use and that it avoids accidentally using env vars from the testing environment.